backend/utils/utils.py

```diff
@@ -4,46 +4,71 @@
 import nltk
 nltk.download('gutenberg')
 from nltk.corpus import gutenberg
+
+
 # Load the KMeans model and TF-IDF vectorizer
 with open('models/kmeans_model.pkl', 'rb') as f:
     loaded_km = pickle.load(f)
 
-with open('models/vectorizer.pkl', 'rb') as f:
+with open('vectorizer.pkl', 'rb') as f:
```

Comment:
why is the path looking outside models folder?

Reply:
@rabelmervin the path should be too model to find the pickle file